pyramid/url.py

```diff
@@ -117,7 +115,7 @@ def _partial_application_url(self, scheme=None, host=None, port=None):
         elif scheme == 'http':
             if port == '80':
                 port = None
-        url += host
+        url = scheme + '://' + host
```

Comment:
Condensing this url assignment into one line from the previous two lines (old lines 99 and 120) is cleaner. :+1: